aptos-move/framework/move-stdlib/sources/acl.move

```diff
@@ -41,6 +41,6 @@ module std::acl {
 
```

Comment:
The `vector::{contains, remove, push_back}` calls above can all be rewritten using receiver-style syntax.